config/routes.rb

```diff
@@ -29,8 +29,8 @@
       get 'issues/:number/details' => 'issues#details'
       post 'issues' => 'issues#create_issue'
       post 'issues/:number/comment' => 'issues#create_comment'
-      put 'issues/comments/:id' => 'issues#update_issue'
-      put 'issues/:number' => 'issues#update_comment'
+      put 'issues/comments/:id' => 'issues#update_comment'
+      put 'issues/:number' => 'issues#update_issue'
```

Comment:
Can't believe I managed to switch those two in both the controller and the routes haha - thanks for catching that.